data-prepper-plugins/kinesis-source/src/main/java/org/opensearch/dataprepper/plugins/kinesis/source/KinesisService.java

```diff
@@ -138,7 +138,7 @@ public Scheduler getScheduler(final Buffer<Record<Event>> buffer) {
             try {
                 scheduler = createScheduler(buffer);
             } catch (Exception ex) {
-                LOG.error("Caught exception when initializing KCL Scheduler. Will retry");
+                LOG.error("Caught exception when initializing KCL Scheduler. Will retry", ex);
```

Comment:
Thanks for adding ex to this message. Also, please add NOISY flag for this message.